docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md

```diff
@@ -81,12 +81,12 @@ Read its [user guide](https://docs.gethue.com/user/concept/#share-to-slack) or t
 
 Get your hands on demo Hue App by:
 
-Running some queries on the [demo live editor](https://demo.gethue.com/).
+- Running some queries on the [demo live editor](https://demo.gethue.com/).
 
-Joining the [demo Slack workspace](https://join.slack.com/t/hue-bot-dev/shared_invite/zt-pqep28s6-zd69W73EOWL5pCVDz06OBg) to see the app in action or by using the following credentials:
+- Logging in to the [demo workspace](https://hue-bot-dev.slack.com/) by using the following credentials:
```

Comment:
--> we should not use the dev slack, need a dedicated one

Reply:
It is the dedicated Demo Slack only, its just bad naming from my side at the starting @romainr 
I'll try renaming it if Slack lets me do it and update the link accordingly.

Reply:
Updated the name and link of the workspace